src/cascadia/TerminalSettingsModel/ActionArgs.cpp

```diff
@@ -586,6 +586,16 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
 
     winrt::hstring GlobalSummonArgs::GenerateName() const
     {
-        return L"";
+        std::wstringstream ss;
+        ss << std::wstring_view(RS_(L"GlobalSummonCommandKey"));
+
+        // "Summon the Terminal window"
+        // "Summon the Terminal window, name:\"{_Name}\""
+        if (!_Name.empty())
+        {
+            ss << L", name: ";
+            ss << std::wstring_view(_Name);
+        }

```

Comment:
I know we've gotten away with it before, but "name" isn't really localized. How about `Summon the "<Name>" window`

Reply:
I'm just gonna punt this discussion to the next PR I've got queued up, because it's gonna become a quagmire of 

* `Summon the Terminal window named "Foo" to the current desktop` (toCurrent/toCurrent/"Foo")
* `Summon the Terminal window on the current desktop` (onCurrent/toCurrent)
* `Summon the Terminal window` (toCurrent/toCurrent)
* `Go to the Terminal window` (any/toCurrent)
* `Summon the Terminal window named "Foo" to the current desktop, to the current monitor`
* `Summon the Terminal window on the current desktop, to the current monitor`
* `Summon the Terminal window on the current desktop, on the current monitor`
* `Summon the Terminal window on the current monitor, to the current desktop` (this one had to change order in English to make sense)

y i k e s